README_ZH.md

```diff
@@ -2,12 +2,26 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-(有些建议在 [go-critic](https://github.com/go-critic/go-critic) 中实现)
-
-- [Go-advices 中文版本](#go-advices-%E4%B8%AD%E6%96%87%E7%89%88%E6%9C%AC)
+- [Go-advice 中文版本](#go-advice-%E4%B8%AD%E6%96%87%E7%89%88%E6%9C%AC)
+    - [Go 箴言](#go-%E7%AE%B4%E8%A8%80)
+    - [Go 之禅](#go-%E4%B9%8B%E7%A6%85)
     - [代码](#%E4%BB%A3%E7%A0%81)
+      - [Always `go fmt` your code.](#always-go-fmt-your-code)
```

Comment:
Thanks for the translation, but I think we should leave Content paragraph only with top level links. What do you think?

Reply:
yes, I fixed.